R/fftrees_grow_fan.R

```diff
@@ -12,20 +23,19 @@ fftrees_grow_fan <- function(x,
   # Prepare: ------
 
   if (!x$params$quiet) {
-    message(paste0("Growing FFTs with ", x$params$algorithm))
+    message(paste0("Growing FFTs with ", x$params$algorithm, ":"))
   }
 
   # Some global variables which could be changed later:
   exit.method <- "fixed"
-  correction <- .25
+  correction  <- .25
```

Comment:
why add spaces here (and below)? I think you are aligning the `<-` vertically but I don't believe this follows tidyverse style guides. I'm ok to leave it but I don't think it's necessary

Reply:
I sometimes align long blocks of assignments to facilitate quick parsing of their left and right parts (as in a table), but have no strong opinions here.  I'd suggest delegating this to some automatic styler in the future.

Reply:
Agreed if indeed `styler::style_pkg()` takes care of this I'd prefer to rely on that

Reply:
Will look into this, but in a separate PR in the future...